worlds/faxanadu/Items.py

```diff
@@ -0,0 +1,50 @@
+from BaseClasses import ItemClassification
+from typing import List 
+
+
+class ItemDef:
+    def __init__(self, id: int, name: str, classification: ItemClassification, count: int, progression_count: int, prefill_location: str):
```

Comment:
```suggestion
    def __init__(self, id: Optional[int, None], name: str, classification: ItemClassification, count: int, progression_count: int, prefill_location: Optional[str, None]):
```
So the typing matches up. Also from typing import Optional

Reply:
Woops, it should be `Optional[int]`, don't put the , None in there